topology/Fermi National Accelerator Laboratory/FNAL USCMS Tier1/USCMS-FNAL-WC1.yaml

```diff
@@ -88,31 +88,6 @@ Resources:
           endpoint: psonar4.fnal.gov
     VOOwnership:
       Fermilab: 100
-  USCMS PerfSonar IPV6 Test:
-    Active: true
-    ContactLists:
-      Administrative Contact:
-        Primary:
-          ID: 86d6367e268ad10200b21b1a85f114e758052fcb
-          Name: Darryl K Wohlt
-      Miscellaneous Contact:
-        Primary:
-          ID: d15cab2ba1421aca11ec451ae36a771822db0778
-          Name: Burt Holzman
-      Resource Report Contact:
-        Primary:
-          ID: 86d6367e268ad10200b21b1a85f114e758052fcb
-          Name: Darryl K Wohlt
-    Description: This is a PerfSonar dual stack instance
-    FQDN: psonartest1.fnal.gov
-    ID: 744
-    Services:
-      net.perfSONAR.Bandwidth:
-        Description: PerfSonar Bandwidth monitoring node
-        Details:
-          endpoint: psonartest1.fnal.gov
-    VOOwnership:
-      Fermilab: 100
```

Comment:
@hyunwoo18 does this look ok to you? `USCMS PerfSonar IPV6 Test` appears to be the exact same resource as [FNAL NRTB PerfSONAR Bandwidth](https://github.com/opensciencegrid/topology/blob/1a6bae8e7f8bab0e713800f6ce4794604bcd0915/topology/Fermi%20National%20Accelerator%20Laboratory/FNAL%20USCMS%20Tier1/USCMS-FNAL-WC1.yaml#L5]

Reply:
Looks ok to me to drop `USCMS PerfSonar IPV6 Test`.

As you mentioned it's identical to `FNAL NRTB PerfSONAR Bandwidth` except that Burt Holzman was listed as a Miscellaneous Contact, and of course the resources had different IDs.

Reply:
Agreed, let's drop the test one